rules/assertion-arguments.js

```diff
@@ -271,6 +278,23 @@ const create = context => {
 
 				checkArgumentOrder({node, assertion: firstNonSkipMember, context});
 			}
+
+			if (gottenArgs === nArgs.max && nArgs.min !== nArgs.max) {
+				let lastArg = node.arguments[node.arguments.length - 1];
+
+				if (lastArg.type === 'Identifier') {
+					const variable = findVariable(context.getScope(), lastArg);
+					let value = null;
+					variable.references.forEach(ref => {
```

Comment:
Use `Array#find` here?

Reply:
I used `forEach` because I want to find the last writing expression. Using `find` would force me to `reverse` the array first.

```js
lastArg = variable.references.reverse().find(ref => ref.writeExpr).writeExpr;
```

Is this really better ?

Reply:
Shouldn't matter. The array is not large. But at least use a `for-of` loop.